GameLibrary/GameLibrary.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <OutputPath>C:\Users\Arnas\source\repos\object-group-game\Server\lib\</OutputPath>
```

Comment:
<OutputPath>$(SolutionDir)\Server\lib\</OutputPath>

Reply:
This was due to Visual Studio's setting, changed it to this